src/components/study/rights/NewStudyRightForm.tsx

```diff
@@ -38,6 +40,20 @@ const NewStudyRightForm = ({ study, user }: Props) => {
     },
   })
 
+  const onEmailChange = (value: string | null) => {
+    form.setValue('email', value || '')
+    form.clearErrors('email')
+    checkExternal(value)
+  }
+
+  const checkExternal = (value: string | null) => {
```

Comment:
```suggestion
  const checkIUserIsExternalUser = (value: string | null) => {
```
et je pense pas que ce soit une fonction utile, tu peux l'inliner (ou alors la mettre dans un useCallback